plugins/copr.py

```diff
@@ -254,6 +269,15 @@ def _remove_repo(cls, repo_filename):
             raise dnf.exceptions.Error(str(e))
 
     @classmethod
+    def _disable_repo(cls, copr_username, copr_projectname):
+        exit_code = call(["dnf", "config-manager", "--set-disabled",
```

Comment:
Maybe you'd like to catch `OSError` or add a `Requires: /usr/bin/dnf` or handle the case when `/usr/bin/dnf` is not installed somehow else...

Anyway, what's wrong with moving the functionality into the "lib" or into DNF's API?